### PR TITLE
Mic-4379/schedule-builds

### PR DESCRIPTION
**Pull request by albrja**
### Adds nightly build and tests to github actions.
- *Category*: Test
- *JIRA issue*: [MIC-4379](https://jira.ihme.washington.edu/browse/MIC-4379)

-Adds nightly schedule build and test run to github actions

### Testing
Will check tomorrow to see if this runs tonight.

### CODE REVIEW

File: .github/workflows/schedule_build.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# -----------------------------------------------------------------------------
+#   - invoked on push, pull_request, or manual trigger
+#   - test under at least 3 versions of python
+# -----------------------------------------------------------------------------
+name: schedule-build
+on:
+  schedule:
+  - cron: "0 0 * * *"
```
Comment (albrja) [inline .github/workflows/schedule_build.yml:8]: Runs at midnight

**Comment by albrja** [comment]
I created this yml file on the github actions page since I didn't think adding the file would create a workflow on github itself. I will need to make sure the action is pointed at main after merging.

File: .github/workflows/schedule_build.yml (hunk 1 above)
Comment (rmudambi) [inline .github/workflows/schedule_build.yml:8]: Is it possible to combine this with the build.yaml by doing something like this?
```
on:
  - push
  - pull_request
  - workflow_dispatch
  - schedule:
    - cron: "0 0 * * *"
```

Comment (albrja) [reply .github/workflows/schedule_build.yml]: That should be possible

**Review by rmudambi — APPROVED** [review]

**Review by mattkappel — APPROVED** [review]

**Pull request merged**